pkg/patch/cmd.go

```diff
@@ -38,7 +39,8 @@ func NewPatchCmd() *cobra.Command {
 				ua.appImage,
 				ua.reportFile,
 				ua.patchedTag,
-				ua.workingFolder)
+				ua.workingFolder,
+				ua.ignoreError)
```

Comment:
please add a test for this if possible

Reply:
applies all the changes below too

Reply:
added in integration tests ([here](https://github.com/project-copacetic/copacetic/pull/247/files#diff-aa5a6d1a6e5802fb07e773e37a06b714eea384d4150f2691586196f3019f3713R61-R72))